server/services/QueueService.js

```diff
@@ -0,0 +1,36 @@
+const log = require('loglevel');
+const queue = require('treetracker-wallet-app/packages/queue');
+const knex = require('../infra/database/knex');
+
+class QueueService {
+  constructor() {
```

Comment:
No need for this constructor, queue and knex can be used directly and don't need to be initialised